generic/tests/cfg/netperf.cfg

```diff
@@ -124,6 +124,23 @@
                     cygwin_start = C:\rhcygwin\Cygwin.bat -i /Cygwin-Terminal.ico -
                     netserv_start_cmd = netserver
                     netperf_install_cmd = cd netperf-2.6.0; ./configure --enable-burst --enable-demo=yes; make; make install
+    variants:
+        # for scenario I440fx/Macvtap/Ovs, could not make them in one loop, should be in separate loop
+        - I440fx:
+            # please add the physical_nic you want to add to your private bridge,like physical_nic = p5p2
+            #physical_nic = p5p2
+        - Macvtap:
+            # please set netdst_nic1 to physica nic like netdst_nic1 = p5p2
+            nettype_nic1 = macvtap
+            #netdst_nic1 =
+            #physical_nic =
+        - Ovs:
+            # If test with ovs scenario. need to set netdst_nic1=[ovs_birdge_name,like ovs0]
+            # netdst_nic1 =
+        - Q35:
+            machine_type = q35
+        - Multi-queue:
+            queues = 4
```

Comment:
Well, only ovs should appear at this variants. Other ones actually do not share the same parameter definitions. i440fx and q35 are existed machine_type variants, directly specify i440fx params like `Linux:` and `Windows:` will be enough. q35 actually specify nothing so should be dropped. macvtap is defined nettype, so same as i440fx. For multi-queue, you should have another variants if need.
```
    i440fx:
         # please add the physical_nic you want to add to your private bridge,like physical_nic = p5p2
         #physical_nic = p5p2
    macvtap:
         # please set netdst_nic1 to physica nic like netdst_nic1 = p5p2
         #netdst_nic1 =
         #physical_nic =
    variants:
         - @default:
         - ovs:
            only bridge ?
             # If test with ovs scenario. need to set netdst_nic1=[ovs_birdge_name,like ovs0]
             # netdst_nic1 =
    variants:
        - @default:
        - multi-queue:
            no ovs ?
            queues = 4
```

Reply:
@yanan-fu Could you help give your opinion on this? Thanks.

Reply:
@vivianQizhu I have update, please help review

Reply:
I still feel weird about these configurations, @yanan-fu Could you help give a look? Thanks.